ocean-client/src/vault-maxi.ts

```diff
@@ -67,7 +67,7 @@ export async function main(event: maxiEvent, context: any): Promise<Object> {
     }
     console.log('using oceans ' + JSON.stringify(oceansToUse))
 
-    const usedLogId = process.env.VAULTMAXI_LOGID ?? settings.logId
+    const usedLogId = settings.logId ?? process.env.VAULTMAXI_LOGID
```

Comment:
hmm no sure what should win. IMHO the envVar is more "direct" so if someone puts in an envVar, it should override the settings.
@Krysh90 what do you think?

Reply:
Problem is, if you do not set the env var, value is set to: "on hostname" in app.ts and the setting logId never wins. This is why I changed it.

Reply:
then lets move this logic to store_config.ts for the local version:
`this.settings.logId = process.env.VAULTMAXI_LOGID ?? this.config.logId ?? 'on ' + os.hostname()` with config.logId being undefined in default

Reply:
thanks for this hint